CMakeLists.txt

```diff
@@ -541,6 +541,46 @@ else()
   message(STATUS "Support for ccache: ${CCACHE_SUPPORT}")
 endif()
 
+if(NOT MSVC)
+  find_program(MOLD_EXECUTABLE mold)
+  default_option(MOLD_SUPPORT "Use 'mold' for linking" "MOLD_EXECUTABLE")
+  if(MOLD_SUPPORT)
+    if(MOLD_EXECUTABLE)
+      message(STATUS "Selecting mold as linker")
+      add_link_options("-fuse-ld=mold")
```

Comment:
Mold support was added in GCC 12.1.0, hence the build will fail with older versions.
I'm not sure if the minimum supported OS versions provide 12.10.0+ or if this section should be guarded by a additional gcc version check.

Reply:
alternative for GCC < 12.10.0:
`-DCMAKE_EXE_LINKER_FLAGS:STRING=-B/usr/bin/mold`
+
`-DCMAKE_LINKER:FILEPATH=/usr/bin/mold`
https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Reducing.20linking.20times/near/377606740
from https://github.com/rui314/mold#how-to-use